buildconfig/class_maker.py

```diff
@@ -184,7 +184,7 @@ def write_test(subproject, classname, filename, args):
   void test_exec()
   {{
     // Create test input if necessary
-    MatrixWorkspace_sptr inputWS = //-- Fill in appropriate code. Consider using TestHelpers/WorkspaceCreationHelpers.h --
+    MatrixWorkspace_sptr inputWS = //-- Fill in appropriate code. Consider using MantidFrameworkTestHelpers/WorkspaceCreationHelper.h --
 
```

Comment:
How is this change related to the current PR?

Reply:
This is an unrelated change I was making previously that snuck in with no relevance, It's still valid though. Can remove it if you want.

Reply:
Ok, no problem, I will approve it